plugins/pre-commit.sh

```diff
@@ -29,38 +29,34 @@ fi
 ########## PIP VERSION #############
 # Verify that yapf is installed; if not, warn and exit.
 if ! command -v yapf >/dev/null; then
-  echo 'yapf not on path; can not format. Please install yapf:'
-  echo '    pip install yapf'
+  printf 'yapf not on path; can not format. Please install yapf:\n' \
+    '    pip install yapf'
   exit 2
 fi
 ######### END PIP VERSION ##########
 
 ########## PIPENV VERSION ##########
 # if ! pipenv run yapf --version 2>/dev/null 2>&1; then
-#   echo 'yapf not on path; can not format. Please install yapf:'
-#   echo '    pipenv install yapf'
+#   printf 'yapf not on path; can not format. Please install yapf:\n' \
+#     '    pipenv install yapf'
```

Comment:
same here -- should just leave it as `echo`

Reply:
reverted back to `echo`